app/lucy/src/app/services/coordinateConversion/location.service.spec.ts

```diff
@@ -77,4 +49,89 @@ describe('ConverterService', () => {
+            fail++;
+            console.log(`\n***`);
+            console.log(`${item.longitude} , ${item.latitude} should be inside: ${item.inside}, but result is: ${isIn}`);
+          } else {
+            success ++;
+          }
+        }
+        console.log(`\n******\nInside Outside:`);
+        console.log(`passed: ${success}`);
+        console.log(`failed: ${fail}`);
+    expect(fail).toBeLessThan(1);
+  });
+  it('should Convert from Lat Long to Albers', () => {
+    const service: ConverterService = TestBed.get(ConverterService);
+    const longitude = -119.472548;
+    const latitude = 49.905577;
+    const albersX = 1468294.70085;
+    const albersY = 564887.24757;
+    const roundTo = 3;
+    const latLongToAlbers = service.latLongCoordinateToAlbers(latitude, longitude)
+    const successful = (latLongToAlbers.x.toFixed(roundTo) === albersX.toFixed(roundTo) && latLongToAlbers.y.toFixed(roundTo) === albersY.toFixed(roundTo));
+    if (!successful) {
+      console.log(`**** latLong To Albers`);
+      console.log(latLongToAlbers);
+    }
+    expect(successful).toBeTruthy();
+  });
+  it('should Convert from Albers to Lat Long', () => {
+    const service: ConverterService = TestBed.get(ConverterService);
+    const longitude = -119.472548;
+    const latitude = 49.905577;
+    const albersX = 1468294.70085;
+    const albersY = 564887.24757;
+    const roundTo = 3;
+    const albersToLatLong = service.albersToLatLongCoordinate(albersX, albersY);
+    const successful = (albersToLatLong.latitude.toFixed(roundTo) === latitude.toFixed(roundTo) && albersToLatLong.longitude.toFixed(roundTo) === longitude.toFixed(roundTo));
+    if (!successful) {
+      console.log(`**** albers To LatLong`);
+      console.log(albersToLatLong);
+    }
+    expect(successful).toBeTruthy();
+  });
+  it('should Convert from lat long to UTM', () => {
+    const service: ConverterService = TestBed.get(ConverterService);
+    const longitude = -119.472548;
+    const latitude = 49.905577;
+    const UTMx = 322462.246733;
+    const UTMy = 5531063.683699;
+    const utmZone = 11;
+    const latLongToUTM = service.convertLatLongCoordinateToUTM(latitude, longitude);
+    const successful = (latLongToUTM.eastings.toFixed() === UTMx.toFixed() && latLongToUTM.northings.toFixed() === UTMy.toFixed() && latLongToUTM.zone === utmZone);
+    if (!successful) {
+      console.log(`**** latLong To UTM`);
+      console.log(latLongToUTM);
+      console.log(`Should be:` + UTMx.toFixed() + ` ` + UTMy.toFixed());
+      console.log(`Is:` + latLongToUTM.eastings.toFixed() + ` ` + latLongToUTM.northings.toFixed());
+    }
+    expect(successful).toBeTruthy();
+  });
+  it('should Convert from UTM to Lat Long', () => {
+    const service: ConverterService = TestBed.get(ConverterService);
+    const longitude = -119.472548;
+    const latitude = 49.905577;
+    const UTMx = 322462.246733;
+    const UTMy = 5531063.683699;
+    const utmZone = 11;
+    const roundTo = 3;
```

Comment:
suggestion for refactoring: create lat, long, UTMx, UTMy, etc. as global variables, since the same values are being reused many times in this file